test_conformance/math_brute_force/binary_double.cpp

```diff
@@ -691,9 +691,11 @@ int TestFunc_Double_Double_Double(const Func *f, MTdata d, bool relaxedMode)
         test_info.k[i].resize(test_info.threadCount, nullptr);
     }
 
-    test_info.tinfo.resize(test_info.threadCount, ThreadInfo{});
+    test_info.tinfo.reserve(test_info.threadCount);
```

Comment:
Did you see an advantage to doing `reserve` then the individual `push_back`s rather than a single `test_info.tinfo.resize(test_info.threadCount)`?

Reply:
The 1-argument overload requires [MoveInsertable](https://en.cppreference.com/w/cpp/named_req/MoveInsertable) and [DefaultInsertable](https://en.cppreference.com/w/cpp/named_req/DefaultInsertable), so yes, that should actually work; thanks for the suggestion!